app/src/main/java/org/schabi/newpipe/report/ErrorActivity.java

```diff
@@ -351,7 +385,8 @@ private String buildJson() {
                     .object()
                     .value("user_action", getUserActionString(errorInfo.userAction))
                     .value("request", errorInfo.request)
-                    .value("content_language", getContentLangString())
+                    .value("content_language", getContentLanguageString())
+                    .value("content_country", getContentLanguageString())
```

Comment:
I think it's a mistake
```suggestion
                    .value("content_country", getContentCountryString())
```